addons/website_slides/views/website_slides_templates_profile.xml

```diff
@@ -50,7 +50,7 @@
 
                             <div class="overflow-hidden mb-1" style="height:24px">
                                 <t t-foreach="course.channel_id.tag_ids" t-as="tag">
-                                    <a t-att-href="'/slides/all?channel_tag_id=%s' % tag.id" t-attf-class="badge o_wslides_channel_tag #{'o_tag_color_'+str(tag.color)}" t-esc="tag.name"/>
```

Comment:
Was it working before your commit ? channel_tag_id does not seem really used ...

Reply:
Nope it wasn't  :)